chevah/keycert/tests/test_ssh.py

```diff
@@ -859,13 +859,14 @@ def test_fromBlob(self):
             common.NS('ssh-dss') + common.MP(2) + common.MP(3) +
             common.MP(4) + common.MP(5))
         dsaKey = keys.Key.fromString(dsaBlob)
-        badBlob = common.NS('ssh-bad')
+        badBlob = common.NS('ssh-\xbd\xbd\xbd')
+        badKey = common.NS('ssh-bad')
         self.assertTrue(rsaKey.isPublic())
         self.assertEqual(rsaKey.data(), {'e': 2L, 'n': 3L})
         self.assertTrue(dsaKey.isPublic())
         self.assertEqual(dsaKey.data(), {'p': 2L, 'q': 3L, 'g': 4L, 'y': 5L})
-        self.assertRaises(
-            keys.BadKeyError, keys.Key.fromString, badBlob)
+        self.assertBadKey(badBlob, 'Invalid non-ascii blob type.')
+        self.assertBadKey(badKey, 'Unknown blob type: \'ssh-bad\'')
```

Comment:
here we have 2 tests. Please break this into 2 tests, each with its own description of the test scenario.

I know that the current tests is not great, but new code should be written based on current best practices